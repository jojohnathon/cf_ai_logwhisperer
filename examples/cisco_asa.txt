%ASA-4-106023: Deny tcp src outside:203.0.113.20/443 dst inside:10.0.0.5/8443 by access-group "outside-in" [0x0, 0x0]
%ASA-4-106023: Deny tcp src outside:203.0.113.21/443 dst inside:10.0.0.5/8443 by access-group "outside-in" [0x0, 0x0]
%ASA-6-302014: Teardown TCP connection 10234 for outside:203.0.113.20/443 to inside:10.0.0.5/8443 duration 0:00:30 bytes 1234 TCP Reset-I
