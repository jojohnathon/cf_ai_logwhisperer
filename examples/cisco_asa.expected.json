{
  "summary": "ASA is dropping outside HTTPS connections to the 8443 service per outside-in policy.",
  "anomalies": [
    "%ASA-4-106023 denies for outside hosts",
    "Connection resets after policy drops"
  ],
  "suggested_commands": [
    { "cmd": "access-list outside-in extended permit tcp host 203.0.113.20 host 10.0.0.5 eq 8443", "why": "Allow trusted host to reach inside service", "risk": "med" },
    { "cmd": "access-list outside-in extended permit tcp host 203.0.113.21 host 10.0.0.5 eq 8443", "why": "Permit additional trusted host", "risk": "med" }
  ]
}
